src/Kongverge/ServiceRegistration.cs

```diff
@@ -40,11 +42,11 @@ public static IServiceCollection ConfigureServices(this IServiceCollection servi
 
                 if (config.Value.DryRun)
                 {
-                    Log.Information("Performing dry run. No writes to Kong will occur.");
+                    Log.Information("Performing dry run: No writes to Kong will occur");
                     return s.GetService<KongAdminDryRun>();
                 }
 
-                Log.Information("Performing live integration. Changes will be made to {host}.", config.Value.Admin.Host);
+                Log.Information($"Performing live integration: Changes will be made to {config.Value.Admin.Host}");
```

Comment:
Do we still get the structured logging with this? I would have thought not - but haven't double-checked.

Reply:
structured logging is a huge benefit when a long-running server app is logging to kibana or similar searchable database.

Kongverge is a console app logging to console.